### ISSUE TITLE
no-vendor-prefixes confuse standard properties and pseudo-elements

### ISSUE THREAD

**Issue by ncoden** [sasstools/sass-lint#953]
I am using `gulp-sass-lint v1.3.2` with `sass-lint v1.10.2`.

For the following code:
```scss
progress {
    &::-ms-fill { ... }
}
```
(from [foundation-sites / forms/_progress.scss](https://github.com/zurb/foundation-sites/blob/546a5b9f38487450df0dcf4957ba8c73bf11194e/scss/forms/_progress.scss#L86))

The following warning is thrown:
```
  86:8  warning  Vendor prefixes should not be used  no-vendor-prefixes
```

The sass-lint config is :
```yml
...
  no-vendor-prefixes:
     - 1
     - ignore-non-standard: true
...
```
(from [.sass-lint.yml](https://gist.github.com/ncoden/ec50bad6cd2f08df0780e9f356442794))

Sass-lint seems (I'm not sure about it) to consider the progress pseudo-element `::-ms-fill` as _standard_ because of the `fill` property. But `::-ms-fill` is specific to IE10 (see https://developer.mozilla.org/fr/docs/Web/CSS/::-ms-fill). I expect `ignore-non-standard` to not check for a standard _pseudo-element_ in a list of _property_.
